### PR TITLE
Updates Docs (CoC) to with a touch more gracious responses in the examples.

### PR DESCRIPTION
**Pull request by banderson623**
I absolutely love these examples. They bring so much context and thought to the codified order of elevation. I added these two small changes to remove "JCF" reference and to a bit more gracious in the face of someone being ridiculous.

I came here from the healthy comments in [Discussion: CoC Violations & Accountability](https://github.com/nodejs/community-committee/issues/111#issuecomment-323868882)

### CODE REVIEW

**Comment by ritcheyer** [comment]
👀

**Comment by ritcheyer** [comment]
👀

**Comment by AgentAntelope** [comment]
I don't think these changes are bad, but I personally think that "being gracious" isn't a requirement in the face of nazis, and I'd like to avoid us implying that it is, or that being a nazi is simply "being ridiculous".

I am not sure how to address that. How do we make clear that graciousness isn't a requirement to getting resolution?

**Comment by scottkellum** [comment]
@AgentAntelope As a wide audience can view discussions I think avoiding language that might be religiously charged, like “JFC”, might be a good idea. This nazi troll clearly has no regard for gracious discourse and it’s important to unequivocally condemn in forceful and no uncertain terms their message I think it’s important to be gracious to the audience witnessing the discussion.

**Comment by AgentAntelope** [comment]
Hrm. Perhaps something like "ABSOLUTELY NOT" then? Expresses the intensity without the religious issue?

**Comment by AgentAntelope** [comment]
I don't feel any need to ask someone calling out a nazi to consider the audience; I don't want to suggest somehow that someone has to be _nice_ whilst alerting people to nazis.

**Comment by rmg** [comment]
FWIW, I like how this reduces the use of slang, regardless of softening/hardening the tone.

**Comment by banderson623** [comment]
I think @rmg nailed what I was attempting to update in the `CODE_OF_CONDUCT.md`. I might have muddled things up in my Pull Request description. Because, I agree with you @AgentAntelope that "being gracious" isn't a requirement. 

I do think that, when possible, kindness is desirable. Even in the most intense debate or moderation, respecting other humans _(even if you don't think they, or their positions, deserve it)_ is a powerful message. But is it required or deserved? Probably not.

**Comment by cjwilburn** [comment]
@rmg I don't think the reduction of the use of slang was a concern, but I agree with @scottkellum's lesser point about no longer using religiously-charged curses. Regarding @AgentAntelope's comment, I agree that I don't think it matters whether or not we're being nice to Nazis, and additionally I think that consideration of the audience witnessing the discussion is borderline concern-trolling, especially if any religiously-charged curses are removed.

**Comment by zkat** [comment]
Gonna xpost what I said on slack here:

I do not think it's necessary nor appropriate to change the tone of this. CoC enforcement is not pretty and it's not comfortable. The entire point of the examples section is to give as realistic a view (from my own actual experience enforcing CoCs like these), of what enforcement looks like in real situations. That involved, very much, conveying the _tone_ of those conversations as effectively as possible.

I'm totally fine with one change, and actually encourage it: we can remove the JFC because frankly, that's not something I should even be doing in general when responding to CoC situations. I don't think it's the end of the world for it to be there, but let's do better anyway.

**Comment by AgentAntelope** [comment]
I'm not putting anything that's intended to respect even a hypothetical nazi into a CoC document.

The only powerful message treating nazis with respect sends is that you consider nazis worth respecting. I do not.

**Comment by banderson623** [comment]
Given your understandable concerns @zkat and @AgentAntelope, should I alter something in this Pull Request? I think my main concern is the religious slang and then we kinda got off track.

**Comment by zkat** [comment]
@banderson623 I think changing the PR such that the only change is "JFC" -> "ABSOLUTELY NOT" (as suggested by @AgentAntelope) will probably get this landed without much further commentary tbh?

**Comment by rmg** [comment]
So it turns out the word I _thought_ was slang is actually a real word and just not one I've ever used or understood: `aggro`.

My point about slang was to ensure the example is accessible to people unfamiliar with the language. I could tell from the context that they were examples of people misbehaving, but I admit I kind of glossed over the actual words and just sort of assumed speaking plain English was part of the implied suggested behaviour.

File: CODE_OF_CONDUCT.md
Comment (zkat) [inline CODE_OF_CONDUCT.md]: could you revert these, as well?

**Comment by AgentAntelope** [comment]
@rmg people can always just ask about language they don't understand :)

**Comment by AgentAntelope** [comment]
I definitely want to make clear that all dialects are equal as far as I'm concerned! Some might just take some learning.

**Comment by banderson623** [comment]
Thanks for the excellent discussion y'all. I think all of the comments have been addressed and the Code of Conduct is better off for it.

Thanks for taking the time to think through all of this with me.

**Comment by zkat** [comment]
@banderson623 thanks for taking the time to open the conversation and working through it! This LGTM now. Since a number of folks commented, I'll wait for their green light to merge+pub

**Review by zkat — APPROVED** [review]
🐑

**Review by AgentAntelope — APPROVED** [review]
👍

**Review by cjwilburn — APPROVED** [review]
👍

**Comment by zkat** [comment]
Alright, that's everyone. I'm merging this in, then.

Thanks, y'all!

**Comment by AgentAntelope** [comment]
@zkat want to merge this, since we're all happy?

**Pull request merged**
